docs/dev.rst

```diff
@@ -1,20 +1,522 @@
+              subfolder: CumulusReports-master/src
+              namespace_strip: npsp
+
+..
+
+    When ``update_dependencies`` runs, the zip is retrieved and the string ``npsp__`` is stripped from all files and filenames in the zip before deployment.  This option is most useful when setting up an unmanaged development environment for an extension package that normally uses managed dependencies.
+    
+    This example takes the NPSP Reports & Dashboards project's unmanaged metadata and strips the references to ``npsp__`` to deploy it against an unmanaged version of NPSP.
+
+
+GitHub Repository Dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+GitHub repository dependencies create a dynamic dependency between the current project and another project on GitHub that uses CumulusCI to manage its dependencies.
+
+    Example: Salesforce EDA
+
+.. code-block:: yaml
+ 
+    project:
+        dependencies:
+            - github: https://github.com/SalesforceFoundation/EDA
+
+When ``update_dependencies`` runs, these steps are taken against the referenced repository.
+
+* Look for the ``cumulusci.yml`` file and parse if found.
+* Determine if the project has subfolders under unpackaged/pre.  If found, deploys them first.
+* Determine if the project specifies any dependencies in the ``cumulusci.yml`` file.  If found, deploys them next in the queue.
+* Determine if the project has a namespace configured in the ``cumulusci.yml`` file. If found, treats the project as a managed package unless the unmanaged option is also ``True``.
+* If the project has a namespace and is not set for unmanaged, use the GitHub API to get the latest release and install it.
+* If the project is an unmanaged dependency, the ``src`` or ``force-app`` directory is deployed.
+* Determine if the project has subfolders under unpackaged/post. If found, deploys them next. Namespace tokens are replaced with ``namespace__`` or an empty string, depending on if the dependency is considered managed or unmanaged.
+
+
+
+Reference Unmanaged Projects
+****************************
+
+If the referenced repository does not have a namespace configured, or if the dependency specifies the ``unmanaged`` option as ``True``, the repository is treated as an unmanaged repository.
+
+    Example: Salesforce EDA
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - github: https://github.com/SalesforceFoundation/EDA
+              unmanaged: True
+
+..
+
+    The EDA repository is configured for a namespace, but the dependency  specifies ``unmanaged: True``, so the dependency deploys unmanaged EDA and its dependencies.
+
+
+
+Reference a Specific Tag
+************************
+
+To reference a version other than HEAD and the latest production release, use the ``tag`` option to specify a tag from the target repository. This option is most useful for testing against beta versions of underlying packages, or recreating specific org environments for debugging.
```

Comment:
HEAD?

Reply:
[HEAD](https://stackoverflow.com/questions/2304087/what-is-head-in-git) is a concept in Git. I think it should be safe to assume that if a user is looking for information on this topic then they should be generally familiar with the term.